### DISCUSSION TITLE
image manager confused about deleted user images

### DISCUSSION THREAD

**Discussion by ssagerian** [AllskyTeam/allsky-modules#65] [General]
I created some images to use with the overlay module. 
Later, I  deleted the image using the image manager , then later still, I see the thumbnail of the image still in the lmage manager, when I try to delete it, while mouse hovering over the image (the image is greyed out), the infomation popup text indicates that the image is in use and can not be deleted. The image is not being used in the overlay. Im using v20230501_3

**Comment by ssagerian** [comment]
The allsky log has a number of the same entry.
ERROR: Cannot locate image random_temperature_humidity_plot.png.
(this is the image I deleted as mentioned earlier)

**Comment by Alex-developer** [comment]
Thanks for the report I have created an issue for it (https://github.com/AllskyTeam/allsky-modules/issues/68) and will look at it under that
